src/rss/rss.ts

```diff
@@ -30,19 +30,19 @@ export default class RSS {
       const feed = await this.parser.parseURL(url);
 
       const converter = new TurndownService();
-      for (const item of feed.items) {
+      for (const item of feed.items ?? []) {
```

Comment:
really? I though `for of` was smarter than needing that

Reply:
I am pretty sure that `for of` doesn't need it, but the strict typing requires it for some reason